libs/ml/include/ml/dataloaders/word2vec_loaders/w2v_dataloader.hpp

```diff
@@ -37,6 +37,8 @@ template <typename T>
 class W2VLoader : public DataLoader<fetch::math::Tensor<T>, fetch::math::Tensor<T>>
 {
 public:
+  static constexpr T WindowContextUnused = -1;
```

Comment:
can you just add a comment here stating what is the expected type T used here? We spoke about this before, and I forget, but if T were SizeType this would roll around to be numeric_max which might be unintended